railties/test/generators/scaffold_generator_test.rb

```diff
@@ -221,7 +221,7 @@ def test_scaffold_on_revoke
     assert_no_file "app/helpers/product_lines_helper.rb"
 
     # Assets
-    assert_file "app/assets/stylesheets/scaffold.css", /:visited/
+    assert_file "app/assets/stylesheets/scaffold.css", /color/
```

Comment:
You shouldn’t change the test here and instead define a :visited style. You are setting a link color (moving away from browser default) so you also should be setting the visited style.